PUBLISHING.md

```diff
@@ -1,6 +1,11 @@
 # Publishing Irving
 These are notes and guidelines for publishing the Irving NPM packages. This will mostly apply to those on the Alley team with NPM credentials.
 
+## Prerequisites
+In order to publish a release you will need to,
+* Have a valid `GH_TOKEN` in your Irving `.env` file. In Github, create your token under [account > Settings > Developer Settings > Personal Access Tokens](https://github.com/settings/tokens). Then create a `.env` file at the root of the cloned Irving repo and assign it to a `GH_TOKEN` environmental variable.
```

Comment:
Maybe add `GH_TOKEN=[your token]` as an example to make it super clear how .env formatting works.